src/include/sof/schedule/task.h

```diff
@@ -91,6 +92,14 @@ static inline uint64_t task_get_deadline(struct task *task)
 	return task->ops.get_deadline(task->data);
 }
 
+static inline bool task_is_ready(struct task *task)
+{
+	if (task->ops.is_ready)
+		return task->ops.is_ready(task->data);
+
+	return true;
+}
+
```

Comment:
@tlauda I do not see any users of is_ready in this series. So maybe we can drop this for now and add when needed?